src/program.js

```diff
@@ -674,6 +674,11 @@ Example: $0 --help run.
         demandOption: false,
         type: 'string',
       },
+      'chromium-debugger-port': {
+        describe: 'Listen port number of the Chromium debugger',
```

Comment:
```suggestion
        describe:
          'An unused port number, for use by the Chromium debugger. ' +
          'If not specified, a random port will be used.',
```

Suggested edit to clarify the meaning, constraints and default behavior.